### DISCUSSION TITLE
Preview with fallback to 200.html locally

### DISCUSSION THREAD

**Discussion by some-user123** [nuxt/nuxt#16867] [Ideas]
After generating a static page (`nuxt generate`), `nuxt preview` will basically run `npx serve -s ./output/public`.
It's really neat being able to test that locally.

However, `serve -s` will fall back to `index.html`, whereas hosting providers typically fall back to `200.html` (which is the whole purpose of this file, right?). Wouldn't it be cool if `nuxt preview` would serve the files with fallback to `200.html`?

Is there an easy way to do it? E.g. `better-serve -s 200.html ./output/public`?

**Comment by pi0** [comment]
Good idea! Why not :) https://github.com/unjs/listhen/issues/38
